media/js/base/mozilla-utils.js

```diff
@@ -50,9 +57,13 @@ if (typeof Mozilla === 'undefined') {
     // then, each key name needs to be preceded by data- as this uses data attributes
     // to work. After this, you can access all strings defined inside the
     // string_data block in JS using Mozilla.Utils.trans('keyofstring'); Thank @mkelly
-    var _$strings = $('#strings');
+    var _strings = document.getElementById('strings');
     Utils.trans = function(stringId) {
-        return _$strings.data(stringId);
+        if (_strings) {
+            return _strings.dataset[stringId];
```

Comment:
Sorry I missed this originally. Using `dataset` here will likely throw errors in older browsers. I'd suggest using `getAttribute` for better legacy support (which is something jQuery did behind the scenes).